docs/02_cleos/03_command-reference/get/table.md

```diff
@@ -3,23 +3,33 @@
 Retrieves the contents of a database table
 
 ## Positional Parameters
-`contract` _TEXT_ - The contract who owns the table
+`account` _TEXT_ - The account who owns the table
```

Comment:
it would be good to mention that the account it is also owning the smart contract code

Reply:
Yes, I think the change from contract to account didn't necessarily make it more clear. I updated as follows:
`account` _TEXT_ - The account who owns the kv_table where the smart contract was deployed